src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=""
+      data-qa="big"
+      method="GET"
+      class="margin"
+    >
+      <label class="position1" for="1">
```

Comment:
Give more clear class names. For example small-input, big-input, input, and label.
Don't use just numbers as an ID. Give proper names.